pyproject.toml

```diff
@@ -24,6 +24,7 @@ pydantic = "*"
 pytest = "*"
 pytest-randomly = "*"
 pytest-cov = "*"
+python-dotenv = "*"
```

Comment:
Do we need a new package for this? I thought that on Grand Challenge we recommend using the poetry plugin for it and then we can read directly from `os.environ.get`?

Reply:
Ah, totally forgot about that combination of poetry + dotenv. I'llfollow that route: way less of an impact, ableit I am surprised by the dependencies it has:

<img width="1470" alt="afbeelding" src="https://github.com/user-attachments/assets/fd2cdec7-3fd9-465e-880b-d7364e4e34ab" />